src/screens/TopicPageScreen/index.js

```diff
@@ -352,7 +362,10 @@ const TopicPageScreen = (props) => {
       const dy = y - lastDragY;
       if (y <= 30) {
         showHeaderAnimation();
-      } else if (dy - 20 > 0) {
+      } else if (
+        dimen.size.TOPIC_CURRENT_ITEM_HEIGHT &&
+        dy - 20 > dimen.size.TOPIC_CURRENT_ITEM_HEIGHT / 2
+      ) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition for triggering the animation has been updated to consider the value of `dimen.size.TOPIC_CURRENT_ITEM_HEIGHT`. This change seems to be aimed at improving the responsiveness of the animation based on the item height.


<!-- This is an auto-generated comment by OSS CodeRabbit -->